modules/contact/include/actions/ContactAction.h

```diff
@@ -98,6 +98,9 @@ class ContactAction : public Action
   /// Whether to use the dual Mortar approach
   bool _use_dual;
 
+  /// Whether to use correct edge dropping treatment
+  bool _correct_edge_dropping;
```

Comment:
This should be made const at some point, but the action needs cleanup to remove old contact options anyways.